kvmagent/kvmagent/plugins/zbs_storage_plugin.py

```
@@ -0,0 +1,71 @@
+        http_server = kvmagent.get_http_server()
+        http_server.register_async_uri(self.CHECK_HOST_STORAGE_CONNECTION_PATH, self.check_host_storage_connection)
+        http_server.register_async_uri(self.UPDATE_HOST_DEPENDENCY_PATH, self.update_host_dependency)
+
+    @kvmagent.replyerror
+    @bash.in_bash
+    def check_host_storage_connection(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
+        rsp = CheckHostStorageConnectionRsp()
+
+        c = 'timeout 20 qemu-io -c "read 0G 4k" -f cbd {}_zbs_:/etc/zbs/client.conf'.format(cmd.path)
+        r, o, e = bash.bash_roe(c)
```

Ruff (0.14.14)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)